src/game/shell.c

```diff
@@ -194,18 +271,40 @@ bool Shell_MakeScreenshot()
         break;
     }
 
-    char path[20];
-    for (int i = 0; i < 10000; i++) {
-        sprintf(path, "screenshot%04d.%s", i, ext);
+    // Screenshot folder
+    const char ss_folder[SS_FOLDER_NAME_SIZE] = "screenshots";
+    char *full_ss_path = NULL;
+    File_GetFullPath(ss_folder, &full_ss_path);
```

Comment:
I think we can just say `File_GetFullPath("screenshots", &full_ss_path)` here and remove `SS_FOLDER_NAME_SIZE` const.